third_party/metadata_envoy/dependencies.json

```diff
@@ -1,5 +1,4 @@
 {
-  "target_path": "/third_party",
   "libraries": [
```

Comment:
Hi, I think you should also include license for envoy itself.

Reply:
Thanks for the catch. Fixed it.